README.md

```diff
@@ -401,6 +402,9 @@ Updating baseboxes
 1. Use `apt-get purge` to remove any old linux-image and linux-header packages to
    save disk space.
 1. Run `vagrant reload` just to make sure grub is still working.
+1. Run
+   `vagrant snapshot push && vagrant ssh --command='sudo /vagrant/test.sh' && vagrant snapshot pop`
+   to ensure that the web server, xdebug, and xhgui are working.
```

Comment:
This takes a long time. Is this a necessary step for updating baseboxes?

Reply:
This failed on my local without logging an error, leaving a bunch of testing data. Perhaps we want to change that second `&&` to `;` so that the snapshot gets restored regardless?

Reply:
![trusty32-lamp zsh 2018-01-04 17-07-54](https://user-images.githubusercontent.com/66118/34586392-cf4d2b8e-f171-11e7-9ec7-0ab97aca80bd.jpg)

Reply:
So it's a _good_ step to ensure everything is working, but not required. Note by "updating baseboxes" we mean maintainers, not individual users.

My original thinking was that if tests failed, you'd want to inspect the state of the VM. Or, is that more confusing?

Reply:
> My original thinking was that if tests failed, you'd want to inspect the state of the VM. Or, is that more confusing?
I think that makes sense, and now that you get an error message on failures, it's less of an issue.